legend-engine-xt-serviceStore-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/from/ServiceStoreGrammarParserExtension.java

```diff
@@ -102,6 +113,37 @@ public Iterable<? extends ConnectionValueParser> getExtraConnectionParsers()
         }));
     }
 
+    @Override
+    public List<Function<SecuritySchemeSourceCode, SecurityScheme>> getExtraSecuritySchemesParsers()
+    {
+        return Lists.mutable.with(code ->
+        {
+            SecuritySchemeParseTreeWalker walker = new SecuritySchemeParseTreeWalker();
+            switch (code.getType())
+            {
+                case "Http":
+                    return parseSecurityScheme(code, p -> walker.visitHttpSecurityScheme(code, p.httpSecurityScheme()));
+                case "ApiKey":
+                    return parseSecurityScheme(code, p -> walker.visitApiKeySecurityScheme(code, p.apiKeySecurityScheme()));
+                default:
+                    return null;
+            }
+        });
+    }
+
+    @Override
+    public List<Function2<String, Map<String,SecurityScheme>, SecuritySchemeRequirement>> getExtraSecurityParsers()
```

Comment:
why we need extension for this?

Reply:
used internally, Lets discuss

Reply:
Have a look at my comment where this is used. We should need to define it here